docs/en/deployment/prepare_deployment_files.md

```diff
@@ -67,19 +67,18 @@ In versions earlier than v3.1.14, v3.2.10, and v3.3.3, StarRocks provides Docker
 1. Download a StarRocks Docker image from [StarRocks Docker Hub](https://hub.docker.com/r/starrocks/artifacts-ubuntu/tags). You can choose a specific version based on the tag of the image.
```

Comment:
Would it be better to use this link instead of pointing directly to the Ubuntu one? https://hub.docker.com/u/starrocks?page=1&search=artifacts